backend.go

```diff
@@ -24,9 +24,6 @@ type Backend interface {
 	// NewIncomingMsg creates a new message from the given params
 	NewIncomingMsg(channel Channel, urn URN, text string) Msg
 
-	// NewOutgoingMsg creates a new outgoing message from the given params
-	NewOutgoingMsg(channel Channel, urn URN, text string) Msg
```

Comment:
This isn't actually needed in the backend interface. (the test backend defines it but it is only used by it)